.github/workflows/main.yml

```diff
@@ -85,6 +86,23 @@ jobs:
           SLACK_USERNAME: RedKubesBot
         if: always() # failure
 
+  integration:
+    runs-on: self-hosted
+    needs: build-test-push-release
+    if: github.event.review.state == 'approved' || contains(github.event.head_commit.message, '[kind]')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: K8s in Docker
+        run: |
+          docker login -u otomi -p $DOCKER_PASSWORD
+          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock --network kind otomi/kind:latest
+          docker cp kind-control-plane:/etc/kubernetes/admin.conf $PWD/kind/kubeconfig
```

Comment:
Does it mean we keep credentials in `kind-control-plane` container? If so then where `kind-control-plane` image is hosted?

Reply:
ok. could you clarify from where the `kind-control-plane`  container comes from ? ( I can't find it in the code)